CMakeLists.txt

```diff
@@ -54,6 +54,10 @@ add_executable(main
     render-pipeline/screen_frame_buffer.h
     helpers/errors.cpp
     helpers/errors.h
+    helpers/logger.cpp
+    helpers/logger.h
+    tests/test.cpp
+    tests/test.h
```

Comment:
we don't need to implement tests in this repo. can you delete this files, please?